test/unit/webdriveragent-specs.js

```diff
@@ -148,6 +148,19 @@ describe('launch', function () {
   });
 });
 
+describe('use wda proxy url', function () {
+  it('should use webDriverAgentUrl wda proxy url', async function () {
```

Comment:
You can remove `async` since your test code does not have await

```
/Users/travis/build/appium/WebDriverAgent/test/unit/webdriveragent-specs.js
  152:52  error  Async function has no 'await' expression  require-await
```

Reply:
Thanks, i fixed this